Cargo.toml

```
@@ -27,6 +28,7 @@ serde_json = {version = "1.0", optional = true}
 sha2 = "0.9"
 stderrlog = {version = "0.5", optional = true}
 structopt = {version = "0.3", optional = true}
+tokio = {version = "1", features=["full"]}
```

[nit] - it takes time to build the whole tokio, we may just import `rt` and then add more features as we need them

---

do you know where i can see a list of all the features? to get the current code to run, I need 
```suggestion
tokio = {version = "1", features=["macros", "rt-multi-thread", "time"]}
```
the only other ones I could find are `rt` and `sync`, but maybe there's a longer list somewhere I can't find